apps/fluent-tester/src/TestComponents/TextExperimental/PressableUsage.tsx

```diff
@@ -33,13 +33,7 @@ export const PressableUsage: React.FunctionComponent = () => {
       <Stack style={stackStyle} gap={5}>
         <Text variant={'bodyStandard'}>
           To learn more about microsoft, visit this{' '}
-          <Text
-            variant={'bodyStandard'}
-            color="blue"
-            keyDownEvents={handledNativeKeyboardEvents}
-            onPress={_onPress}
-            onKeyDown={_onKeyDown}
-          >
+          <Text variant={'bodyStandard'} color="blue" keyDownEvents={handledNativeKeyboardEvents} onPress={_onPress} onKeyDown={_onKeyDown}>
```

Comment:
Nit: use `useKeyProps` instead of passing in `onKeyDown / keyDownEvents` manually hook so that onKeyDown would (eventually) also work on macOS